docker/android-sdk/tools/watch

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+inotifywait -mr $HOME/.android -e create -e moved_to -e modify -e attrib |
```

Comment:
or we upload it somewhere (like jenkins) and download/copy it as part of the build process.